libafl_cc/build.rs

```diff
@@ -234,8 +234,14 @@ pub const LIBAFL_CC_LLVM_VERSION: Option<usize> = None;
     let llvm_bindir = exec_llvm_config(&["--bindir"]);
     let bindir_path = Path::new(&llvm_bindir);
 
-    let clang = bindir_path.join("clang");
-    let clangcpp = bindir_path.join("clang++");
+    let clang_path = env::var_os("CLANG").unwrap();
```

Comment:
Are these always set? If not, we should not `unwrap` but fall back to the original names, instead

Reply:
Good point, it should be the same as with LLVM_CONFIG

Reply:
Ok I have discovered another problem. The ``bindir_path`` variable is also used to build the passes, so even if we change the env variables, ``bindir_path`` wouldn't change. It didn't want to build previously because it didn't want to find "clang" because of the missing ".exe" extension. Therefore, I have added a config check (cf. https://github.com/AFLplusplus/LibAFL/pull/1237/commits/14bca18b94ab154541d394a03eaae48c4c6abee3) and also added the exe extension in the ``build_pass`` function for the windows config. 
Now, however, the user has to have ``llvm-config``, ``clang.exe`` and ``clang++.exe`` in the same directory (which should be default anyways... So the immediate solution, is to copy the clang files from the bin directory to wherever ``llvm-config`` is. I hope this doesn't break anything, but for now it builds without a problem).